components/announcement_bar/configuration_bar/configuration_bar.tsx

```diff
@@ -25,6 +26,7 @@ import warningIcon from 'images/icons/warning-icon.svg';
 
 import UserProfile from 'components/user_profile/user_profile';
 import RenewalLink from '../renewal_link/';
+import PurchaseNowLink from 'components/announcement_bar/purchase_now_link/purchase_now_link';
```

Comment:
nit: Simplify the name of this component.
```suggestion
import PurchaseLink from 'components/announcement_bar/purchase_link/purchase_link';
```